roles/common/tasks/main.yml

```diff
@@ -17,7 +17,7 @@
       - python-pycurl
       - python-psycopg2
       - python3-psycopg2
-      - python-software-properties
```

Comment:
Can we make this change only for Ubuntu 18, maybe with `when: ansible_distribution_major_version == '18'`? I'm worried we break our current setups when merging this to master.

Reply:
Yeah, we could do that. I see this dependency was added in 2014, and probably to support Ansible. I'm not sure it's needed any more?

Reply:
I'll take a look at extracting these bits out into a separate role where the compatibility-specific code can live.

Reply:
Done.

Reply:
I just found an Ansible note here: https://docs.ansible.com/ansible/2.4/intro_installation.html#latest-releases-via-apt-ubuntu that says:

`On older Ubuntu distributions, “software-properties-common” is called “python-software-properties”.`

It looks like it was around in Ubuntu 12, but has been slowly deprecated since then.